content/en/docs/ops/configuration/traffic-management/dns-proxy/snips.sh

```diff
@@ -93,8 +99,59 @@ kubectl exec deploy/sleep -- curl -sS -v auto.internal
 *   Trying 240.240.0.1:80...
 ENDSNIP
 
+snip_external_tcp_services_without_vips_1() {
+kubectl create ns external-1
+kubectl -n external-1 apply -f samples/tcp-echo/tcp-echo.yaml
+}
+
+snip_external_tcp_services_without_vips_2() {
+kubectl create ns external-2
+kubectl -n external-2 apply -f samples/tcp-echo/tcp-echo.yaml
+}
+
+snip_external_tcp_services_without_vips_3() {
+kubectl apply -f - <<EOF
+apiVersion: networking.istio.io/v1beta1
+kind: ServiceEntry
+metadata:
+  name: external-svc-1
+spec:
+  hosts:
+  - tcp-echo.external-1.svc.cluster.local
+  ports:
+  - name: external-svc-1
+    number: 9000
+    protocol: TCP
+  resolution: DNS
+---
+apiVersion: networking.istio.io/v1beta1
+kind: ServiceEntry
+metadata:
+  name: external-svc-2
+spec:
+  hosts:
+  - tcp-echo.external-2.svc.cluster.local
+  ports:
+  - name: external-svc-2
+    number: 9000
+    protocol: TCP
+  resolution: DNS
+EOF
+}
+
+snip_external_tcp_services_without_vips_4() {
+istioctl pc listener deploy/sleep | grep 9000
+}
+
+! read -r -d '' snip_external_tcp_services_without_vips_4_out <<\ENDSNIP
+240.240.105.94 9000  ALL                                                                                           Cluster: outbound|9000||tcp-echo.external-2.svc.cluster.local
```

Comment:
This line is very long and hard to read. I wonder if there's a better way to display?